tests/acceptance/index-test.js

```diff
@@ -219,4 +218,24 @@ describe('Acceptance | index', function() {
     expect(find('[data-test-weekly-overview-day=29].holiday')).to.have.length(1)
     expect(find('[data-test-weekly-overview-day=28].holiday')).to.have.length(0)
   })
+
+  it('rollbacks the absence modal', async function() {
+    await visit('/?day=2017-06-29')
+
+    await click('[data-test-add-absence]')
+
+    await click('[data-date=2017-06-30]')
```

Comment:
I was a little confused reading this. Wouldn't it be more logical if you'd move this await below the first expect call?
Also, I would have found it more logical to expect the 29th to be selected in both cases, instead of expecting the 30th not to be selected in the last expect call.

Reply:
Seems fair.. Fixed!